### ISSUE TITLE
Version 6.4.1 fails to create simulator artifact when using SPM

### ISSUE THREAD

**Issue by jzeisweiss** [Iterable/swift-sdk#532]
**NOTES**: 
- Your derived data must be clean. If you previously built successfully inside Xcode you can not recreate this issue. It only happens on a clean build.
- This is very similar to https://github.com/Iterable/swift-sdk/issues/523 but specifically for simulators.
- Reverting to version 6.3.4 will successfully generate a simulator artifact.
- I'm using Xcode 13.2.1
- I'm using Swift Package Manager

When I attempt to archive my app for a simulator, to upload to Saucelabs for automated UI testing, the build fails.

Command: `xcodebuild -scheme Pinseeker -project ./Project.xcodeproj -configuration Adhoc -derivedDataPath scan_derived_data -destination 'platform=iOS Simulator,name=iPhone 13' -archivePath archives/simulator_build/Project.simulator\ 2022-02-10\ 17.43.43.xcarchive archive`

**Errors**:
```
The following build commands failed:
	CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler (in target 'IterableSDK' from project 'IterableSDK')
	CompileSwiftSources normal i386 com.apple.xcode.tools.swift.compiler (in target 'IterableSDK' from project 'IterableSDK')
	CompileSwift normal i386 (in target 'IterableSDK' from project 'IterableSDK')
	CompileSwiftSources normal arm64 com.apple.xcode.tools.swift.compiler (in target 'IterableSDK' from project 'IterableSDK')
```

```
❌  /Users/me/GitHub/pinseeker-ios/scan_derived_data/spm/checkouts/swift-sdk/swift-sdk/swiftui/InboxViewRepresentable.swift:22:40: cannot find type 'Context' in scope

    func makeUIViewController(context: Context) -> IterableInboxViewController {
                                       ^~~~~~~



❌  /Users/me/GitHub/pinseeker-ios/scan_derived_data/spm/checkouts/swift-sdk/swift-sdk/swiftui/InboxViewRepresentable.swift:34:91: cannot find type 'Context' in scope

    func updateUIViewController(_ uiViewController: IterableInboxViewController, context: Context) {
                                                                                          ^~~~~~~



❌  /Users/me/GitHub/pinseeker-ios/scan_derived_data/spm/checkouts/swift-sdk/swift-sdk/swiftui/InboxViewRepresentable.swift:11:32: cannot find type 'UIViewControllerRepresentable' in scope

struct InboxViewRepresentable: UIViewControllerRepresentable {
                               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~



❌  /Users/me/GitHub/pinseeker-ios/scan_derived_data/spm/checkouts/swift-sdk/swift-sdk/swiftui/IterableInboxView.swift:73:27: cannot find type 'View' in scope

    public var body: some View {
                          ^~~~



❌  /Users/me/GitHub/pinseeker-ios/scan_derived_data/spm/checkouts/swift-sdk/swift-sdk/swiftui/IterableInboxView.swift:11:34: cannot find type 'View' in scope

public struct IterableInboxView: View {
```

**Comment by mrtnrst** [comment]
This error is because of #524, which adds the `!arch(arm)` into the include. A probable cause is the current iOS SDK-supported version, which builds `armv7`, causing the error in building/archiving. Could the SwiftUI inbox be moved to an example app for users of SwiftUI as opposed to including it in the package itself? Otherwise, I would assume there is still more work to ensure the SDK builds appropriately for all users.

**Comment by tapashmajumder** [comment]
@mrtnrst We will take a look at this soon.

**Comment by tapashmajumder** [comment]
Should be fixed with version 6.4.2. Please open a ticket with your CSM if you are still seeing issues.
